### PR TITLE
Upgrade to Terraform 1.0

### PR DESCRIPTION
**Pull request by xingao267**

### CODE REVIEW

File: examples/tfengine/generated/org_foundation/cicd/README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ in [./configs](./configs) directory to point to your container.
 
 We leverage Google Container Registry's Vulnerability scanning feature to detect
 potential security risks in this container, and
-[None](https://console.cloud.google.com/gcr/images/cloud-foundation-cicd/global/cft/developer-tools-light@sha256:d881ce4ff2a73fa0877dd357af798a431a601b2ccfe5a140837bcb883cd3f011/details?tab=vulnz)
+[None](https://console.cloud.google.com/gcr/images/cloud-foundation-cicd/global/cft/developer-tools-light@sha256:7c40255f4df73cca81c943456bf1ebe6c58da455a24d90b5d7d9964e97bf5646/details?tab=vulnz)
```
Comment (MartinPetkov) [inline examples/tfengine/generated/org_foundation/cicd/README.md:70]: I don't see a "vulnz" tab at this link. Is that intentional?

**Review by MartinPetkov — COMMENTED** [review]
Do we also need to set `required_version` to `>=1.0.1` in the examples/ dir?

File: examples/tfengine/generated/org_foundation/cicd/README.md (hunk 1 above)
Comment (xingao267) [reply examples/tfengine/generated/org_foundation/cicd/README.md:70]: It's at the end after the question mark

File: build/presubmit_int.yaml [hunk 2]
```diff
@@ -12,14 +12,19 @@
...
 
 timeout: 3600s
 
 options:
-  env: ["GOPATH=/go"]
+  env:
+  - "GOPATH=/go"
+  # Unset TF_PLUGIN_CACHE_DIR so different build steps do not share Terraform cache.
+  # It's set to `/workspace/test/integration/tmp/.terraform` in the CFT Developer Tool's container
+  # for CFT's testing purpose. We don't need it.
```
[the remark this answers is not in the document]
Comment (xingao267) [reply build/presubmit_int.yaml:26]: Setting this actually cause problems as our tests that run in the different steps should not interfere with each other. The previous container does not set this. It's a change on the CFT container side with this new container. I just unset it to preserve the existing behavior.

[the remark this answers is not in the document]
Comment (MartinPetkov) [reply build/presubmit_int.yaml:26]: Oh I see. That makes sense.

File: examples/tfengine/generated/org_foundation/cicd/README.md (hunk 1 above)
Comment (xingao267) [reply examples/tfengine/generated/org_foundation/cicd/README.md]: Discussed offline.

**Comment by MartinPetkov** [comment]
Do we also need to set required_version to >=1.0.1 in the examples/ dir?

**Comment by xingao267** [comment]
> Do we also need to set required_version to >=1.0.1 in the examples/ dir?

Just updated. Set it to >=1.0

**Review by MartinPetkov — APPROVED** [review]

**Pull request merged**